crates/iota-framework/packages/iota-system/sources/voting_power.move

```diff
@@ -31,26 +31,46 @@ module iota_system::voting_power {
     const EVotingPowerOverThreshold: u64 = 3;
     const EInvalidVotingPower: u64 = 4;
 
+    const ECommitteeMembersSetCorrupt: u64 = 102;
+
+
     /// Set the voting power of all validators.
     /// Each validator's voting power is initialized using their stake. We then attempt to cap their voting power
     /// at `MAX_VOTING_POWER`. If `MAX_VOTING_POWER` is not a feasible cap, we pick the lowest possible cap.
-    public(package) fun set_voting_power(validators: &mut vector<ValidatorV1>) {
+    public(package) fun set_voting_power_v1(validators: &mut vector<ValidatorV1>) {
```

Comment:
why do we need this v1?

Reply:
This is used during IotaSystemStateV1 during genesis creation.